commands/slashReload.js

```diff
@@ -8,26 +8,40 @@ module.exports = {
     "hideHelp": true,
     run: async (bot, args, message) => {
         if (!bot.owners.includes(message.author.id)) return;
-        const msg = await message.channel.send("Zaczynam proces przeładowania!");
+        const errors = [], completed = [];
+        const generateEmbed = () => {
+            return new Discord.MessageEmbed()
```

Comment:
A nie dało by rade tutaj użyć bot.embed?

Reply:
można ale to jest customowy embed do komendy deweloperskiej nie związany z innymi komendami, poza tym potem zmienia się kolor ze względu na wynik komendy

Reply:
No okej ale kolor embeda można nadpisać a tak to masz jednego require mniej :lenny: